src/IssueViz.Security.UnitTests/Store/HotspotsStoreTests.cs

```diff
@@ -228,6 +263,20 @@ public void GetLocations_NoHotspotsForGivenFilePath_EmptyList()
             locations.Should().BeEmpty();
         }
 
+        [TestMethod]
+        [DataRow(null)]
+        [DataRow("")]
+        public void GetLocations_HotspotHasNoFilePath_EmptyList(string filePath)
```

Comment:
If this is a regression test for the bug then it can be useful to label it as such.